### PR TITLE
feat: Add listItemBookmark component

### PR DESCRIPTION
**Pull request by shawncothran**
change props to match OP API
(#385)

### CODE REVIEW

**Comment by thomasthesecond** [comment]
Can we change src/components/listItem**Bookable**/index.jsx to src/components/listItem**Bookmark**/index.jsx

File: src/components/bookmarkButton/index.jsx [hunk 1]
```diff
@@ -27,9 +27,9 @@ const iconProps = {
   style: { display: "block" },
 };
 
-const Bookmark = ({ onClick, size, marked, style }) => (
+const BookmarkButton = ({ onClick, size, marked, style }) => (
```
Comment (thomasthesecond) [inline src/components/bookmarkButton/index.jsx]: I kinda thing this change should be its own pull request

Comment (shawncothran) [inline src/components/bookmarkButton/index.jsx]: done

Comment (thomasthesecond) [inline src/components/bookmarkButton/index.jsx]: 🙏
